lighthouse-core/test/report/report-test.js

```diff
@@ -57,10 +57,26 @@ describe('Report', () => {
     const html = reportGenerator.generateHTML(sampleResults);
 
     assert.ok(html.includes('<footer'), 'no footer tag found');
-    assert.ok(html.includes('printButton = document.querySelector'),
-              'lighthouse-report.js was not inlined');
-    assert.ok(html.includes('.report-body {'), 'report.css was not inlined');
-    assert.ok(!html.includes('&quot;lighthouseVersion'), 'lhresults were not escaped');
+    assert.ok(html.includes('window.lhresults = '), 'report results were inlined');
```

Comment:
do you want to test more than this? (though you do test for `lighthouseVersion`, below?) In the past we had a failure for a while where `window.lhresults = undefined;` was passing the test